tools/scalar-schema/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM clojure:openjdk-11-lein-slim-buster AS BUILD
+COPY . /code
+WORKDIR /code
+RUN lein uberjar
```

Comment:
I think we should use the released version instead of building it itself like `scalardl-schema-loader` because the main is development branch.
https://github.com/scalar-labs/scalardl-schema-loader/blob/master/Dockerfile

Reply:
I actually first wanted to push this PR into the 3.1 branch for this exact reason. The idea is to build and publish the image based on the Dockerfile in a branch of a released version, not from master.

Reply:
Ah, it's in the same repo as scalardb, so that it should be fine.